fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRDbConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.connection;
+
+
+/**
+ * Constants used with FHIR DB datasources and connections
+ * 
+ * @implNote extracted from FHIRDbDAO
+ */
+public class FHIRDbConstants {
+    public static final String FHIRDB_JNDI_NAME_DEFAULT = "jdbc/fhirProxyDataSource";
+    public static final String PROPERTY_DB_DRIVER = "dbDriverName";
+    public static final String PROPERTY_DB_URL = "dbUrl";
+    public static final String PROPERTY_DB2_USER = "user";
```

Comment:
```suggestion
    public static final String PROPERTY_DB_USER = "user";
```

Reply:
agreed, although these were taken directly from the old FHIRDbDao.java.